client/pages/tracks.jsx

```diff
@@ -0,0 +1,171 @@
+    });
+  }
+
+  componentDidMount() {
+    fetch('/api/tracksalltime')
+      .then(res => res.json())
+      .then(userInfo => {
+        this.setState({
+          displayName: null,
+          timeRange: null,
+          clickedOn: null,
+          track: userInfo
+        });
+      });
+  }
+
+  render() {
+    if (this.state.track === undefined) {
+      return null;
+    }
+    return (
+      <div className='home-container'>
+        <div id="header-container">
+          <div id="header">TOP ARTISTS</div>
+          <div id="message">Hi Justin, here are your top artists of all time.</div>
+        </div>
+        <div id="time-container" className='padding-top'>
+          <div className='column-one-third padding-left'>
+            <div className="time-range"> All Time </div>
+          </div>
+          <div className='column-one-third'>
+            <div className="time-range"> 6 Months </div>
+          </div>
+          <div className='column-one-third padding-right'>
+            <div className="time-range"> 1 Month </div>
+          </div>
+        </div>
+        <div id="tracks-container">
+          <div className='row padding-top'>
+            <div className='column-one-half-row padding-left'>
+              <div className='column-one-half align-center'>
+                <img className='track-cover' src={this.state.track.key0.image} />
+              </div>
+              <div className='column-one-half'>
+                <p className='track-info track-name margin-none'>{this.state.track.key0.track}</p>
+                <p className='track-info'>{this.state.track.key0.artist}</p>
+                <p className='track-info'>Popularity: {this.state.track.key0.popularity}</p>
+              </div>
+            </div>
+            <div className='column-one-half-row padding-right'>
+              <div className='column-one-half align-center'>
+                <img className='track-cover' src={this.state.track.key1.image} />
+              </div>
+              <div className='column-one-half'>
+                <p className='track-info track-name'>{this.state.track.key1.track}</p>
+                <p className='track-info'>{this.state.track.key1.artist}</p>
+                <p className='track-info'>Popularity: {this.state.track.key1.popularity}</p>
+              </div>
+            </div>
+          </div>
+          <div className='row padding-top'>
+            <div className='column-one-half-row padding-left'>
+              <div className='column-one-half align-center'>
+                <img className='track-cover' src={this.state.track.key2.image} />
+              </div>
+              <div className='column-one-half'>
+                <p className='track-info track-name'>{this.state.track.key2.track}</p>
+                <p className='track-info'>{this.state.track.key2.artist}</p>
+                <p className='track-info'>Popularity: {this.state.track.key2.popularity}</p>
+              </div>
+            </div>
```

Comment:
This is a crazy amount of copy-pasted code. Try refactoring it to use `.map()` once you fix your back end code.

You can make one "row" div with a `flex-wrap: wrap;` and then each track can go inside of a `column` div. if needed, you can put a row/column combination _inside_ of the track column.

But this big bunch of tracks jsx code should only really be about 20 lines total.